app/src/test/java/org/mozilla/fenix/home/sessioncontrol/SessionControlViewTest.kt

```diff
@@ -90,6 +91,53 @@ class SessionControlViewTest {
         assertFalse(state.shouldShowHomeOnboardingDialog(settings))
     }
 
+    @Test
+    fun `GIVEN recent tabs disabled in settings WHEN calling shouldShowRecentTabs THEN section should not be shown`() {
```

Comment:
Depending if we decide to move this to ext/AppState, i would also move the tests as well. 

One minor nit is avoiding the `shouldShowRecentTabs` method name usage in the test name - https://github.com/mozilla-mobile/fenix/pull/23010#discussion_r802089174. I am gonna start a wiki that will document these micro-decisions that sometimes happen in code reviews.

Reply:
Yeah, I agree. It looked like the existing convention within that test suite, but I'll update it when I move it.